torch/_inductor/codecache.py

```diff
@@ -1223,25 +1204,17 @@ def iterate_over_candidates() -> Generator[CompiledFxGraph, None, None]:
                 if len(meta.cached_kernel_names) > 0:
                     get_metrics_context().increment("num_triton_bundles", 1)
 
-        inductor_meta = autotune_cache.inductor_meta_from_config()
-        AutotuneCacheBundler.begin_compile(inductor_meta, code=code)
-
         try:
-            with dynamo_timed(
-                "PyCodeCache.load_by_key_path", log_pt2_compile_event=True
-            ):
-                graph.current_callable = PyCodeCache.load_by_key_path(
-                    graph.cache_key,
-                    artifact_path,
-                    graph.cache_linemap,
-                    graph.get_constants(gm),
-                ).call
+            artifact_path = graph.after_deserialization(gm)
         except OSError:
             # Not expected, but in case the PyCodeCache entry is removed from
             # underneath us, treat it as a cache miss and recompile.
-            log.error("Failed to load cached artifact: %s", artifact_path)
```

Comment:
Removing the log line seems worse

Reply:
The log still happens - just inside `after_deserialization` which will raise an exception (because it doesn't know if ignoring the exception is the right thing or not).